StereoKitC/systems/render.cpp

```diff
@@ -28,7 +28,13 @@
 #include "../libraries/stb_image_write.h"
 #pragma warning(pop)
 
-#include <DirectXMath.h> // Matrix math functions and objects
+// Matrix math functions and objects
+#if defined(_WIN32)
```

Comment:
For Linux specifically, I might prefer SK_OS_LINUX from platform_utils.h, otherwise I'll also have to check this for Android/Web.

Reply:
done